todo_app/data/trello_items.py

```diff
@@ -0,0 +1,61 @@
+import os, requests
+from todo_app.data.classes.Item import Item
+
+base_url = "https://api.trello.com/1/"
+
+headers = {
+  "Accept": "application/json"
+}
+
+query = {
+  'key': os.getenv('TRELLO_API_KEY'),
+  'token': os.getenv('TRELLO_API_TOKEN')
+}
+
+board_id = os.getenv('TRELLO_BOARD_ID')
+to_do_list_id = os.getenv("TRELLO_TO_DO_LIST_ID")
+done_list_id = os.getenv("TRELLO_DONE_LIST_ID")
+
+def get_items():
+    """
+    Fetches all saved items from the Trello board.
+
+    Returns:
+        list: The list of saved items.
+    """
+    url = base_url + f"boards/{board_id}/lists"
+    lists = requests.request("GET", url, headers=headers, params={**query, **{"cards": "open"}}).json()
+    return [Item.from_trello_card(card, list) for list in lists for card in list["cards"]]
```

Comment:
Nice use of this one liner, I'll admit I think it's complicated enough to at least justify some line-breaks to spell it out though!
```python
    return [
            Item.from_trello_card(card, list) 
            for list in lists
            for card in list["cards"]
    ]
```